Makefile

```diff
@@ -0,0 +1,11 @@
+provision:
+	./tools/laptop_setup.sh
+
+hugo-start:
```

Comment:
Based on the slack thread, I think @alekseymykhailov decided that we'll do the static render from hugo and serve via nginx.